esdm/esdm_es_irq.c

```diff
@@ -217,7 +217,7 @@ static int esdm_irq_seed_monitor(void)
 	if (!esdm_config_es_irq_entropy_rate())
 		return 0;
 
-	if (ent >= esdm_config_es_irq_entropy_rate()) {
+	if (ent >= esdm_config_es_irq_entropy_rate() + esdm_compress_osr()) {
```

Comment:
Why is this? The kernel IRQ code already reduces the entropy rate.

Reply:
This is just in the notification path, to prevent reads from kernel, where entropy level is too less.